hw/top_earlgrey/cdc/cdc_waivers.gpio.tcl

```diff
@@ -3,7 +3,12 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 # Verix CDC waiver file
+# Expression:
+#  ControlSignal==""
+#  ReconSignal==""
+#  MultiClockDomains=="IO_DIV2_CLK::IO_DIV4_CLK"
 
-set_rule_status -rule { W_RECON_GROUPS } -status { Waived } \
-   -expression {u_gpio.gen_filter} \
-   -comment {filters are reconverged into registers. But it's OK.}
\ No newline at end of file
+set_rule_status -rule { W_RECON_GROUPS } -status { Waived }     \
+   -expression {(ControlSignal=~"*u_gpio.gen_filter*") &&       \
+                (ReconSignal=~"*u_gpio.u_reg.u_intr_state.q*")} \
+   -comment {filters are reconverged into registers. But it's OK.}
```

Comment:
Ah is this the filter output eventually converging into an interrupt going out that something happened?

Reply:
Yep. filtering happens in DIV2 and the signal is converged into the status for edge event. PADRING/PINMUX use IO_DIV2 clock that's why I think this reconvergence error occurs.

Reply:
sounds good.

Reply:
Could we just say something along the lines of... the convergence is for status reporting, the events are really all independent?

Reply:
OK Let me quickly revise it